day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,48 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_reader :base, :toppings
+  attr_accessor :protein
+  
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    toppings.push(topping)
+  end
+
+  def remove_topping(topping)
+    toppings.delete(topping)
+  end
+
+  def change_protein(protein)
+    self.protein = protein
+  end
```

Comment:
Another way of updating the attributes directly is to use the instance variable `@protein` (or `@toppings`).